core/src/test/java/io/confluent/kafka/schemaregistry/ClusterTestHarness.java

```diff
@@ -118,10 +117,12 @@ public ClusterTestHarness(int numBrokers, boolean setupRestApp, String compatibi
   @Before
   public void setUp() throws Exception {
     zookeeper = new EmbeddedZookeeper();
-    zkConnect = String.format("127.0.0.1:%d", zookeeper.port());
+    zkConnect = String.format("localhost:%d", zookeeper.port());
     zkUtils = ZkUtils.apply(
         zkConnect, zkSessionTimeout, zkConnectionTimeout,
-        JaasUtils.isZkSecurityEnabled());
+        false); // true of false doesn't matter because the schema registry Kafka principal is the same as the
```

Comment:
Add: because all services (kafka, schema registry etc) run in a single process in the unit tests, ...